tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -1906,9 +1932,130 @@ void SwiftLangSupport::getCursorInfo(
     return;
   }
 
+  /// Counts how many symbols \p Res contains.
+  auto ResultCount = [](const RequestResult<CursorInfoData> &Res) -> size_t {
+    if (Res.isCancelled()) {
+      return 0;
+    } else if (Res.isError()) {
+      return 0;
+    } else {
+      return Res.value().Symbols.size();
+    }
+  };
+
+  /// Serializes \c CursorInfoData into a string.
+  auto ResultDescription =
+      [](const RequestResult<CursorInfoData> &Res) -> std::string {
+    if (Res.isCancelled()) {
+      return "cancelled";
+    } else if (Res.isError()) {
+      return Res.getError().str();
+    } else {
+      std::string Description;
+      llvm::raw_string_ostream OS(Description);
+      // The ContainerTypeUSR varies too much between the solver-based and
+      // AST-based implementation. A few manual inspections showed that the
+      // solver-based container is usually more correct than the old. Instead of
+      // fixing the AST-based container type computation, exclude the container
+      // type from the verification.
```

Comment:
Where is the container type USR being filtered out?

Reply:
In the next PR (not open yet). I removed the comment for now.